modules/tinymce/src/core/main/ts/ForceBlocks.ts

```diff
@@ -88,7 +87,7 @@ const addRootBlocks = (editor: Editor) => {
 
       if (!rootBlockNode) {
         rootBlockNode = dom.create(forcedRootBlock, Options.getForcedRootBlockAttrs(editor));
-        node.parentNode.insertBefore(rootBlockNode, node);
+        rootNode.insertBefore(rootBlockNode, node);
```

Comment:
Another minor code change here as `node.parentNode` is nullable per the TS dom types. Since we know that parent is `rootNode` here and it's not nullable we can use it instead.